src/lcm/simulate.py

```diff
@@ -65,6 +69,8 @@ def simulate(
             )
         vf_arr_list = solve_model(params)
 
+    params = converter.params_to_internal(params)
```

Comment:
```suggestion
    internal_params = converter.params_to_internal(params)
```
Let's not overwrite inputs / have variables contain different but similar things at different points in a function.